lib/opal/parser.rb

```diff
@@ -3,6 +3,7 @@
 require 'opal/ast/builder'
 require 'opal/rewriter'
 
+# rubocop:disable Naming/UncommunicativeMethodParamName
```

Comment:
Is there a way to avoid these directives? they're so ugly 🙀

Reply:
Probably we could extract it to a separate file and disable it for this cop using `.rubocop.yml`